src/serialport.cpp

```diff
@@ -681,6 +731,7 @@ extern "C" {
     Nan::SetMethod(target, "open", Open);
     Nan::SetMethod(target, "update", Update);
     Nan::SetMethod(target, "write", Write);
+    Nan::SetMethod(target, "read", Read);
```

Comment:
or maybe I could keep it in a 'common' cpp file. I could avoid using an ifdef if there was something like a SetExtraMethods() in the init function where native bindings could add extra methods (in this case read) or just do nothing. What do you think?